extra/docker/base/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Based on Ubuntu
 ############################################################
 
-FROM ubuntu:jammy
+FROM ubuntu:noble
```

Comment:
I'd expect building the Docker images to fail now due to removal of python2 packages in 24.04. That won't be an issue when we drop Python2 support ourselves in #2519